blocks/identity-block/features/account-management/_children/SocialEditableFieldContainer.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { PrimaryFont } from '@wpmedia/shared-styles';
+import styled from 'styled-components';
+import getThemeStyle from 'fusion:themes';
+import { useFusionContext } from 'fusion:context';
+import './social-editable-field-styles.scss';
+
+const StyledContainer = styled.div`
+  border-color: ${(props) => props.borderColor};
+`;
+
+// todo: add disconnect and connect funcs
+function SocialEditableFieldContainer({ foundUsername, children, identityType }) {
+  // if username found then show the username?
+  const [isConnected, setIsConnected] = useState(Boolean(foundUsername));
```

Comment:
if falsy foundUsername, then isConnected should be false. although the foundUsername really should be there regardless. I guess isConnected could probably be passed in as props then or a derived value rather than state?

Reply:
updated